### DISCUSSION TITLE
Definicion del Proyecto Imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo2#129] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by WayarCarla** [comment]
Buenas noches profesor creamos en el repositorio la carpeta con nombre proyecto Imposible , en la misma subimos al igual que el proyecto auto las fichas correspondientes la de revisión del proyecto,  diagrama en bloque , diagrama de gantt, vídeo del grupo donde definimos el tema a desarrollar  . A parte de eso debemos subir algo más ? Por ahí interpretamos que debíamos trabajar de forma similar al proyecto del auto. 
Saludos Grupo 2

**Reply by Gona79** [reply]
Que tal Carla, esta muy bien. Es justamente eso lo que tenian que hacer. saludos.

**Comment by Gona79** [comment]
**Chicos como estan:** 

- [x] Manejo de Control de versiones: Ok
- [x] Manejo de Estructura de proyecto:  Ok
- [ ] Presentacion Final: Falta desarrollar el prototip final y la presentacion.

**Comment by JorEl057** [comment]
Devolución Grupal: @dr-arriola  @WayarCarla  @Jerec4stro  @OscarAGazzola  @danydeitu 
Nota Final Grupal: 7 (Siete)

Ideación del Proyecto: 7 (Siete)
Presentación: 6 (Seis)
Uso de recursos: 10 (Diez)
Desarrollo: 6 (Seis)
Implementación: 6 (Seis)

Nota Final Grupal: 7 (Siete)

Estimados tomaron la idea de Iluminación Inalámbrica como proyecto y la ampliaron como un proyecto general de uso para poblaciones rurales, aplicable en cualquier zona con características similares. Esto demuestra su compromiso y RSU. En su presentación que cumplieron correctamente con un tiempo acorde, hablan sobre la necesidad en forma general y la implementación en forma particular. Trabajaron demostrando una organización y de recursos agiles muy bien logrados. Finalmente el desarrollo y la implementación del proyecto también fueron logrados. Felicitaciones y continúen trabajando así!!!
